docs/user-guide/cli-using-editing-team-configuration.md

```diff
@@ -0,0 +1,43 @@
+# Editing team configurations
+
+After you [initialize team configuration](../user-guide/cli-using-initializing-team-configuration), the newly created team profiles need additional details in order to be applied in your environment. This could include information such as a port number, or user credentials.
+
+You might also need to modify the configuration file to add new profiles for accessing mainframe services.
```

Comment:
It might be good to link to the "Creating profiles" article here, or maybe we should add more information about how users can add profiles. I think most users can figure out how to modify existing properties, but its not really clear how users can determine what kinds of profiles they can add and what the available properties for a given profile are.

The JSON Intellisense/autocompletion in VS Code helps a lot since it shows all of the profile types and available properties as defined in the schema, but if you are using a basic text editor I'm not sure how you would know what to do. There might not be a good answer here.

Reply:
Will address this at a later time. Created [Issue 3526](https://github.com/zowe/docs-site/issues/3526) for tracking purposes.